app/views/support_interface/regions/edit.html.erb

```diff
@@ -5,29 +5,30 @@
 <%= form_with model: [:support_interface, @region] do |f| %>
   <%= f.govuk_error_summary %>
 
+  <%= render "shared/support_interface/forms/application_form", f: %>
   <%= f.govuk_check_box :application_form_skip_work_history, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Skip work history" } %>
-  <%= f.govuk_check_box :reduced_evidence_accepted, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Accept reduced evidence" } %>
-
-  <%= f.govuk_collection_select :sanction_check, [
-    OpenStruct.new(id: 'online', name: 'Online'),
-    OpenStruct.new(id: 'written', name: 'Written'),
-    OpenStruct.new(id: 'none', name: 'None')
-  ], :id, :name, label: { text: "Sanction check" } %>
-
-  <%= f.govuk_collection_select :status_check, [
-    OpenStruct.new(id: 'online', name: 'Online'),
-    OpenStruct.new(id: 'written', name: 'Written'),
-    OpenStruct.new(id: 'none', name: 'None')
-  ], :id, :name, label: { text: "Status check" } %>
-
-  <%= render "shared/teaching_authority_form_fields", f: %>
+  <%= render "shared/support_interface/forms/preliminary_check_form_fields", f: %>
+  <%= render "shared/support_interface/forms/eligibility_checker", f: %>
+  <%= render "shared/support_interface/forms/status_sanctions_checks", f: %>
+  <%= render "shared/support_interface/forms/teaching_authority_form_fields", f: %>
+
+  <%= f.govuk_collection_radio_buttons :teaching_authority_requires_submission_email,
+  [
+  OpenStruct.new(value: :false, label: "Yes"),
+  OpenStruct.new(value: :true, label: "No")
+  ],
+  :value,
+  :label,
+  legend: { text: "Automatically notify the teaching authority (via email) every time a QTS application is submitted?" } %>
+
+  <%= render "shared/support_interface/forms/changes_to_eligible_page", f: %>
+  <%= render "shared/support_interface/forms/qualifications_information_form_fields", f: %>
 
-  <%= f.govuk_check_box :teaching_authority_provides_written_statement, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Teaching authority will only send written statement directly to TRA" } %>
+  <%= f.govuk_check_box :reduced_evidence_accepted, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Accept reduced evidence" } %>
   <%= f.govuk_check_box :written_statement_optional, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Applicant can submit without uploading the written statement (if a written statement is requested)" } %>
+  <%= f.govuk_check_box :teaching_authority_provides_written_statement, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Teaching authority will only send written statement directly to TRA" } %>
```

Comment:
I think the text for this checkbox should be:

> Will the teaching authority only send the letter of professional standing (LOPS) directly to the TRA?

I also think it's supposed to go nearer the top of the page.

https://lucid.app/teamspaces/8063bc26-a127-483b-bdfc-c7264994a5f8/edit?invitationId=inv_a6622e61-1fd0-4a59-af86-0db455de7bde#